### PR TITLE
Documentation: Update screenshots and docs about hybrid app.

### PR DESCRIPTION
**Pull request by niley7464**
This patch updates some screenshots and documentation.
Also, it provides github action workflow usage for easy test.

### CODE REVIEW

File: documentation/getting_started.md [hunk 1]
```diff
@@ -243,6 +243,37 @@ WARNING: Default profile is used for sign. This signed package is valid for emul
...
+$ cd nnstreamer-example/Tizen.web/ImageClassificationOffloading
+$ tizen package -t wgt -r ../ImageClassificationOffloadingService/Debug/EQmf4iSfpX-1.0.0-arm.tpk  -- .buildResult/ImageClassificationOffloading.wgt
+The active profile is used for signing. If you want to sign with other profile, please use '--sign' option.
+...omitted...
+Package( nnstreamer-example/Tizen.web/ImageClassificationOffloading/.buildResult/ImageClassificationOffloading.wgt ) is created successfully.
+```
+Tizen Web App build process is done automatically through github actions.
+So if you don't need to build it yourself, you can download it from this [link](https://github.com/nnstreamer/nnstreamer-example/actions/workflows/tizen-build.yml).
+
+Click `Run workflow` with main branch.
```
Comment (niley7464) [inline documentation/getting_started.md:269]: https://github.com/nnstreamer/nnstreamer-example/actions/workflows/tizen-build.yml

Run workflow buttons from workflow_dispatch is only visiable to who has a permission now.

Comment (wooksong) [reply documentation/getting_started.md:269]: Can we then make a pre-built package for this?

File: documentation/getting_started.md [hunk 2]
```diff
@@ -645,13 +676,14 @@ successfully installed and launched on the device.
 ### Run Tizen-to-Android Offloading
 
 To run Tizen-to-Android Offloading you need to connect both devices to the same Wi-Fi network or
-ethernet. Press the start button on the Android device to run the server. Then, the port number the
-server is listening will display on the screen like the following screenshot.
+ethernet. Press the load models button on the Android device to run the offloading service
+based on the conf files. Then, the port number which the service is listening will display
```
Comment (wooksong) [inline documentation/getting_started.md:680]:
```suggestion
based on the conf files. Then, the port number to which the service is listening will display
```

File: documentation/getting_started.md (hunk 1 above)
Comment (niley7464) [reply documentation/getting_started.md:269]: Yes.
I just added workflow-dispatch option to the tizen-build workflow.
https://github.com/nnstreamer/nnstreamer-example/pull/358

A permission change is required to allow all users to trigger this workflow.

File: documentation/getting_started.md (hunk 2 above)
Comment (niley7464) [reply documentation/getting_started.md]: Updated! Thank you

File: documentation/getting_started.md (hunk 1 above)
Comment (wooksong) [reply documentation/getting_started.md:269]: @niley7464 https://github.com/orgs/nnstreamer/packages?repo_name=nnstreamer-example

Comment (niley7464) [reply documentation/getting_started.md:269]: Is the pre-built package needed because action artifacts can expire?

Comment (wooksong) [reply documentation/getting_started.md:269]: @niley7464 I mean, if specific permission is required to run the workflow, why don't we just provide a link to the action artifacts or example's repository package?

File: documentation/getting_started.md [hunk 3]
```diff
@@ -243,6 +243,37 @@ WARNING: Default profile is used for sign. This signed package is valid for emul
 Package File Location: nnstreamer-example/Tizen.web/ImageClassificationOffloading/.buildResult/ImageClassificationOffloading.wgt
 ```
 
+We need a hybrid app to get the port value automatically from the offloading service.
```
Comment (gichan-jang) [inline documentation/getting_started.md:246]: What is the hybrid app?
I think it will be helpful if there is more explanation.

File: documentation/getting_started.md (hunk 1 above)
Comment (niley7464) [reply documentation/getting_started.md:269]: I thought it should be able to check the build progress at certain points in time.
If a pre-built package is okay, I will add action artifacts link.

File: documentation/getting_started.md [hunk 4]
```diff
@@ -645,13 +676,14 @@ successfully installed and launched on the device.
...
 
 ![Android Device (Start the server): A-34-arm64](img/A-34-arm64_port.jpg)
 
-Use android's port number in Tizen application and create pipelines by pressing buttons. When the
-image classification task runs successfully, the detected label and inference time are displayed on
+Tizen devices automatically find offloading services using the model ID. When the image
+classification task runs successfully, the detected label and inference time are displayed on
 the screen as shown below.
 
 ![Tizen Device (Start the client): T-8.0-armv7l](img/T-8.0-armv7l_run.png)
```
Comment (gichan-jang) [inline documentation/getting_started.md:689]: This comment is not directly related with this change, but it seems that there are too many meaningless decimal points for inference time in picture.
IMO, three decimal points are enough.

File: documentation/getting_started.md (hunk 1 above)
Comment (wooksong) [reply documentation/getting_started.md:269]: Good to go:)

File: documentation/getting_started.md (hunk 4 above)
Comment (niley7464) [reply documentation/getting_started.md:684]: I agree
https://github.com/nnstreamer/nnstreamer-example/pull/360
Added this PR and updates screenshot

Thank you :)

File: documentation/getting_started.md (hunk 3 above)
Comment (niley7464) [reply documentation/getting_started.md:247]: Added what is hybrid application.
Thank you

File: documentation/getting_started.md (hunk 1 above)
Comment (niley7464) [reply documentation/getting_started.md]: Added documentation that users can download applications via github workflow artifacts.

**Review by gichan-jang — APPROVED** [review]
LGTM

**Review by wooksong — APPROVED** [review]
